stealth.lua

```diff
@@ -0,0 +1,107 @@
+
+cloaking.privs_to_string = minetest.privs_to_string
+function minetest.privs_to_string(privs, delim)
+    assert(type(privs) == "table")
+    delim = delim or ","
+    local list = {}
+    for priv, bool in pairs(privs) do
+        if bool and priv ~= "cloaking" then
+            list[#list + 1] = priv
+        end
+    end
+    return table.concat(list, delim)
+end
+
+
+local privs_command = minetest.registered_chatcommands["privs"].func
+minetest.override_chatcommand("privs", {
+    func = function(caller, param)
+        if minetest.check_player_privs(caller, "cloaking") or minetest.check_player_privs(caller, "admin") then
```

Comment:
Where did the `admin` privilege come from?

Reply:
Apparently a hallucination. I thought that was a priv when I wrote this, I wonder if I got it confused w/ "server" or some admin-only priv. I put it in there, as you mentioned that the "stealth" feature could confuse admins. I can either remove the reference, or change it to another suitable priv if you think there is one.